tests/unit/test_MultiContainerInterface.py

```diff
@@ -1,4 +1,4 @@
-import unittest2 as unittest
+import unittest as unittest
```

Comment:
```suggestion
import unittest
```

Reply:
looks good though! Thanks for cleaning up that python 2 clutter!

Reply:
Thanks! Somehow missed that.